docs/releases/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: "Release notes"
+has_children: true
+nav_order: 4
+has_toc: false
+---
+
+# Release notes
+
+GlareDB is open source and contains [full release notes on GitHub].
+
+- [August 2023]
+  - [Initial Iceberg support](/docs/releases/2023-august.html#initial-iceberg-support)
+  - [Create native table from query](/docs/releases/2023-august.html#create-native-table-from-query)
+  - [Improvements to SQL functions](/docs/releases/2023-august.html#improvements-to-sql-functions)
+  - [Exclude columns in `SELECT`](/docs/releases/2023-august.html#exclude-columns-in-select)
+  - [Misc updates and fixes](/docs/releases/2023-august.html#misc-updates-and-fixes)
```

Comment:
these hrefs are inlined because there will likely be similar headings as time goes on and more months are added.